extensions/swagger-ui/deployment/pom.xml

```diff
@@ -30,6 +30,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-openapi-common-deployment</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jackson</artifactId>
```

Comment:
You need to include the `-deployment` dependency here and the runtime one in the runtime pom.